corehq/apps/fixtures/views.py

```diff
@@ -570,6 +571,8 @@ def _get_fixture_upload_args_from_request(request, domain):
             replace = True
         elif replace.lower() == "false":
             replace = False
+        user_email = request.couch_user.email if request.couch_user.email is not None \
+            else request.couch_user.username
```

Comment:
Is it intentional that all users are going to get this email? Should this be controlled by a feature flag or only go to dimagi emails? Not sure how many people use this API,

Reply:
How about the SUPPORT feature flag? Not all of QA have dimagi emails or are superusers.

Reply:
I liked the idea of putting this behind the `SUPPORT` feature flag, and I've updated this PR to reflect that